GliaWidgets/Lib/Presenter/GliaPresenter.swift

```diff
@@ -0,0 +1,34 @@
+import UIKit
+internal final class GliaPresenter {
+    private var window: UIWindow? {
+        if #available(iOS 13, *) {
+            return UIApplication.shared.windows.first { $0.isKeyWindow }
+        } else {
+            return UIApplication.shared.keyWindow
+        }
+    }
+
+    internal var root: UIViewController {
```

Comment:
Let's improve the naming for this property. I think the `root` is not the best name for it because of
- the class name is `***Presenter`, so, at least for me, this property name looks like it should be the `root` (or parent) presenter or smth like that. If you would like, let's name it `rootViewController` at least.
- it has `root` name but returns the opposite view controller - the most presented view controller. So, perhaps we could put the previous name, smth like `topMostViewController`

What do you think, does my concern make sense?

Reply:
I like to think of it as root for presenter, so root that the presenter will present on, basically. But it's a good argument

Reply:
I think we can make it private also